src/domain/app/login/LoginPage.tsx

```diff
@@ -1,7 +1,7 @@
 import { Button, useOidcClient } from 'hds-react';
 import * as React from 'react';
 import { useTranslation } from 'react-i18next';
-import { useLocation } from 'react-router';
+import { useLocation } from 'react-router-dom';
```

Comment:
Here useLocation has been fixed to be consistently imported from react-router-dom.